src/main/java/org/datadog/jenkins/plugins/datadog/DatadogBuildListener.java

```diff
@@ -200,8 +201,34 @@ public final void onCompleted(final Run run, @Nonnull final TaskListener listene
           // The client is a threadpool so instead of creating a new instance of the pool
           // we lease the exiting one registerd with Jenkins.
           statsd = getDescriptor().leaseClient();
-          statsd.incrementCounter("completed", tagsToCounter);
-          logger.fine(String.format("Attempted to send 'completed' counter with tags: %s", Arrays.toString(tagsToCounter)));
+          statsd.incrementCounter("completed", statsdTags);
+          logger.fine(String.format("Attempted to send 'completed' counter with tags: %s", Arrays.toString(statsdTags)));
+
+          logger.fine("Computing KPI metrics");
+          // Send KPIs
+          if (run.getResult() == Result.SUCCESS) {
+            long mttr = getMeanTimeToRecovery(run);
+            long cycleTime = getCycleTime(run);;
+            long leadTime = run.getDuration() + mttr;
+
+            statsd.gauge("leadtime", leadTime / 1000, statsdTags);
```

Comment:
Shouldn't the name be prefixed with `jenkins.job`?

Reply:
When we lease the statsd client, we setup the prefix then - https://github.com/DataDog/jenkins-datadog-plugin/blob/46b025d16bb1b8725664aa0f822a77ded5b2d592/src/main/java/org/datadog/jenkins/plugins/datadog/DatadogBuildListener.java#L404